src/net/sourceforge/kolmafia/request/PlaceRequest.java

```diff
@@ -343,6 +375,34 @@ private static void parseSotVisit(String responseText) {
     }
   }
 
+  // visible for testing
+  /**
+   * This takes a string from KoL that represents an available location where the character can
+   * adventure. If the input uniquely matches a KoLmafia location then the matching location is
+   * returned. Otherwise it is checked against a static mapping of KoL strings that could be mapped
+   * into one of several KoLmafia locations. If the character status allows exactly one of these
+   * locations to be adventured in then that location is returned.
+   *
+   * @param location - A string from KoL that represents a unique and available adventure location.
+   * @return - A KoLAdventure matching the unique and available location or null
+   */
+  protected static KoLAdventure getAdventerableLocation(String location) {
```

Comment:
`getAdventurableLocation`

but otherwise looks reasonable.

Reply:
Changed.  Doesn't seem to be in a US dictionary but neither is adventerable and I like this one better.  Thanks.